src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerTest.java

```diff
@@ -1,27 +1,33 @@
 package uk.gov.hmcts.reform.wacaseeventhandler.controllers;
 
+import org.junit.After;
 import org.junit.Test;
 import org.springframework.http.HttpStatus;
 import uk.gov.hmcts.reform.wacaseeventhandler.SpringBootFunctionalBaseTest;
 import uk.gov.hmcts.reform.wacaseeventhandler.domain.EventInformation;
 
 import java.time.LocalDateTime;
+import java.util.UUID;
 
 import static net.serenitybdd.rest.SerenityRest.given;
+import static org.hamcrest.CoreMatchers.is;
 import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
 import static uk.gov.hmcts.reform.wacaseeventhandler.CreatorObjectMapper.asJsonString;
 
 public class CaseEventHandlerControllerTest extends SpringBootFunctionalBaseTest {
 
+    private final String caseReference = UUID.randomUUID().toString();
+    private String taskId;
+
     @Test
```

Comment:
Do we need a test to cover the invalidEventInformation scenario? I recall we discussed this but couldn't spot the test in FT on IT?